libvast/vast/system/signal_monitor.hpp

```diff
@@ -43,7 +43,7 @@ class signal_monitor {
   static auto run_guarded(std::thread& t, caf::actor_system& sys,
                           std::chrono::milliseconds monitoring_interval,
                           caf::actor receiver) {
-    t = std::thread{[&] {
+    t = std::thread{[=, &sys, receiver{std::move(receiver)}] {
```

Comment:
I find `receiver=std::move(receiver)` less noisy, but I'm curious why you chose brace-initialization.

Reply:
So vertical whitespace is the devil but horizontal is ok? I didn't make a conscious decision here.